package.json

```diff
@@ -40,6 +40,8 @@
     "electron-simple-updater": "^1.5.0",
     "electron-squirrel-startup": "^1.0.0",
     "font-awesome": "^4.7.0",
+    "jquery": "^3.4.1",
```

Comment:
are these (jquery and popper) to aid the review mod stuff, or to fix a bootstrap visual bug?

Reply:
They are both dependencies for bootstrap javascript code for the dropdown menu.